server.js

```diff
@@ -13,4 +13,40 @@ app.listen(5000, () => {
   console.log('Example app listening on port 5000!');
 });
 
+// MIDDLEWARE
+// success: next()
+// error: res.json({error: "You did wrong!"})
+
+//1 
+// 404 handler => kicks in if we did not found any matching route handler
+/* app.use( (req, res, next) => {
+  res.status(404).json({ error: `Route ${req.url} does not exist`})
+}) */
+
+//2
+/* app.get("/fuckup/saw-it-coming", (req, res, next) => {
```

Comment:
That route seems to be fine, could be commented in :) 

You just need to produce some error in the try { ... } block. Otherwise the catch block would never get called.